Withings-SDK-iOS/WithingsMeasureAPIClient.m

```diff
@@ -234,8 +261,18 @@ + (MeasuresDateRange*)dateRangeBetweenStartDate:(NSDate*)startDate andEndDate:(N
 {
     NSDate *startDate = _startDate ? _startDate : [NSDate dateWithTimeIntervalSince1970:0];
     NSDate *endDate = _endDate ? _endDate : [NSDate date];
-    return @{@"startdate" : @([startDate timeIntervalSince1970]),
```

Comment:
@jdrevet Of course. I had thought that I had done this but it seems not. I'll do that and also check everything through again. Apologies for the oversight.